lib/jnpr/junos/facts/swver.py

```diff
@@ -80,12 +80,15 @@ def __ne__(self, other):
 def _get_swver(dev, facts):
     # See if we're VC Capable
     if facts['vc_capable'] is True:
-        return dev.rpc.cli("show version all-members", format='xml')
-    else:
         try:
-            return dev.rpc.cli("show version invoke-on all-routing-engines", format='xml')
```

Comment:
This should not have changed.  That original command supports both QFX/EX and MX.  Please revert.